src/v/kafka/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ v_cc_library(
     v::bytes
     v::rpc
     v::cluster
+    v::coproc
```

Comment:
This should probably be coupled with a change to remove `v::coproc` from `v/redpanda/CMakeLists.txt`

Reply:
Ah @dotnwat pointed out that this introduces a circular dependency. This PR no longer needs this to be included since the issue was with `describe_configs_resource_result` which is no longer redacted. So I just removed this, and we can untangle the dependencies separately